bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-toolbar/component.jsx

```diff
@@ -283,15 +283,15 @@ class WhiteboardToolbar extends Component {
      * 4. Trigger initial animation for the icons
     */
     // 1st case
-    if (this.thicknessListIconRadius && this.thicknessListIconColor) {
+    if (　(this.thicknessListIconRadius && this.thicknessListIconColor) || annotationSelected.value === 'text') {
       if (colorSelected.value !== prevState.colorSelected.value) {
```

Comment:
Another solution is to completely remove this condition, which I confirmed to work as well.